client/lib/post-normalizer/rule-content-disable-autoplay.js

```diff
@@ -1,22 +1,17 @@
-/** @format */
-
 /**
  * External dependencies
  */
-
-import { forEach } from 'lodash';
+import { forEach, isArray } from 'lodash';
 import url from 'url';
 
-/**
- * Internal Dependencies
- */
-
 function stripAutoPlays( query ) {
 	const keys = Object.keys( query ).filter( function( k ) {
 		return /^auto_?play$/i.test( k );
 	} );
 	forEach( keys, key => {
-		const val = query[ key ].toLowerCase();
+		// In the rare case that we're handed an array of values, use the first one
+		const firstValue = isArray( query[ key ] ) ? query[ key ][ 0 ] : query[ key ];
```

Comment:
I think you could just use `Array.isArray` and avoid the lodash dependency.

Reply:
updated in https://github.com/Automattic/wp-calypso/pull/32096/commits/d2155786275d37c4db34d6bd427ad115587b4646